webapi/webapi-twoinone-xwalk-tests/README.md

```diff
@@ -0,0 +1,25 @@
+# RealSense Test Suite
+
+## Introduction
+
+This test suite is for checking TwoInOne support in Crosswalk Project:
+* https://github.com/crosswalk-project/crosswalk-extensions-twoinone/blob/master/twoinone-windows/XWalkExtensionApi.js
+* JIRA Ticket: https://crosswalk-project.org/jira/browse/XWALK-5711

```

Comment:
I find the feature is still Open, what's the latest status? How could your tests pass?